tests/Doctrine/Tests/Common/Cache/CacheProviderTest.php

```diff
@@ -143,4 +143,23 @@ public function testInvalidNamespaceVersionCacheEntry() : void
 
         self::assertTrue($cache->deleteAll());
     }
+
+    public function testFetchWillNotResultInMissingCacheKeyAfterFirstTry(): void
```

Comment:
Can you rename the method so it exposes what you are fixing? The method body seems to attempt two `doFetch` oprations and a `doContains`, but no namespace-related operations seem to be happening.